backend/worker/requirements.txt

```diff
@@ -4,3 +4,5 @@ mitmproxy==5.2
 cryptography==2.9
 pytest==6.0.1
 scrapy==2.3.0
+dnspython>=1.14.0
+dnstwist
```

Comment:
Pin the dependency

Reply:
Can we do this? something like:
```suggestion
dnstwist==20201228
```

Also, do we need to explicitly specify the dnspython dependency or is it already included when you add dnstwist?